src/file_pipe_log/format.rs

```diff
@@ -20,6 +21,43 @@ const LOG_REWRITE_SUFFIX: &str = ".rewrite";
 /// File header.
 const LOG_FILE_MAGIC_HEADER: &[u8] = b"RAFT-LOG-FILE-HEADER-9986AB3E47F320B394C8E84916EB0ED5";
 
+pub(crate) fn is_valid_paddings(buf: &[u8]) -> Result<bool> {
+    let len = buf.len();
```

Comment:
Let's change it to test the first and last u8 instead. return buf.is_empty() || buf[0] == 0 && buf[buf.len() - 1] == 0